wezterm-gui/src/overlay/copy.rs

```diff
@@ -627,8 +634,20 @@ impl CopyRenderable {
         }
     }
 
+    fn get_pattern(&self) -> Pattern {
+        return match self.pattern_type {
+            PatternType::CaseSensitiveString => {
+                Pattern::CaseSensitiveString(self.search_line.get_line().to_owned())
+            }
+            PatternType::CaseInSensitiveString => {
+                Pattern::CaseInSensitiveString(self.search_line.get_line().to_owned())
+            }
+            PatternType::Regex => Pattern::Regex(self.search_line.get_line().to_owned()),
+        };
```

Comment:
`return` is redundant, and let's factor out the common pattern string bit:

```suggestion
        let pattern = self.search_line.get_line().to_string();
        match self.pattern_type {
            PatternType::CaseSensitiveString => {
                Pattern::CaseSensitiveString(pattern)
            }
            PatternType::CaseInSensitiveString => {
                Pattern::CaseInSensitiveString(pattern)
            }
            PatternType::Regex => Pattern::Regex(pattern),
        }
```

Reply:
Fixed